sdk/videoanalyzer/video-analyzer-edge/samples/typescript/src/lvaInvokeModuleSample.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  PipelineTopology,
+  Request,
+  RtspSource,
+  UnsecuredEndpoint,
+  NodeInput,
+  LivePipeline,
+  createRequest,
+  IotHubMessageSink
+} from "@azure/video-analyzer-edge";
+
+import { Client } from "azure-iothub";
+
+function buildPipelineTopology() {
+  const rtspSource: RtspSource = {
+    name: "rtspSource",
+    endpoint: {
+      url: "${rtspUrl}",
+      "@type": "#Microsoft.VideoAnalyzer.UnsecuredEndpoint",
+      credentials: {
+        username: "${rtspUserName}",
+        password: "${rtspPassword}",
+        "@type": "#Microsoft.VideoAnalyzer.UsernamePasswordCredentials"
+      }
+    } as UnsecuredEndpoint,
+    "@type": "#Microsoft.VideoAnalyzer.RtspSource"
+  };
+
+  const nodeInput: NodeInput = {
+    nodeName: "rtspSource"
+  };
+
+  const msgSink: IotHubMessageSink = {
+    name: "msgSink",
+    inputs: [nodeInput],
+    hubOutputName: "${hubSinkOutputName}",
+    "@type": "#Microsoft.VideoAnalyzer.IotHubMessageSink"
+  };
+
+  const pipelineTopology: PipelineTopology = {
+    name: "jsTestTopology",
+    properties: {
+      description: "description for jsTestTopology",
+      parameters: [
+        { name: "rtspUserName", type: "String", default: "dummyUsername" },
+        { name: "rtspPassword", type: "SecretString", default: "dummyPassword" },
+        { name: "rtspUrl", type: "String" },
+        { name: "hubSinkOutputName", type: "String" }
+      ],
+      sources: [rtspSource],
+      sinks: [msgSink]
+    }
+  };
+
+  return pipelineTopology;
+}
+
+function buildLivePipeline(pipelineTopologyName: string) {
+  const livePipeline: LivePipeline = {
+    name: "jsLivePipelineTest",
+    properties: {
+      description: "description",
+      topologyName: pipelineTopologyName,
+      parameters: [{ name: "rtspUrl", value: "rtsp://sample.com" }]
+    }
+  };
+
+  return livePipeline;
+}
+
+export async function main() {
+  const device_id = "lva-sample-device";
+  const module_id = "mediaEdge";
+  const connectionString = "connectionString";
+  const iotHubClient = Client.fromConnectionString(connectionString);
+
+  const invokeMethodHelper = async (methodRequest: Request<any>) => {
```

Comment:
I think this provides slightly better typing?

```suggestion
  async function invokeMethodHelper<T>(methodRequest: Request<T>) {
```